.gitpod.Dockerfile

```diff
@@ -1,42 +1,4 @@
-# Based on https://github.com/gitpod-io/template-nix/blob/main/.gitpod.Dockerfile
-
-FROM gitpod/workspace-base
-
-USER root
-
-# Install Nix
-RUN addgroup --system nixbld \
-  && adduser gitpod nixbld \
-  && for i in $(seq 1 30); do useradd -ms /bin/bash nixbld$i &&  adduser nixbld$i nixbld; done \
-  && mkdir -m 0755 /nix && chown gitpod /nix \
-  && mkdir -p /etc/nix && echo 'sandbox = false' > /etc/nix/nix.conf
-  
-# Install Nix
-CMD /bin/bash -l
-USER gitpod
-ENV USER gitpod
-WORKDIR /home/gitpod
-
-RUN touch .bash_profile \
- && curl https://nixos.org/releases/nix/nix-2.15.0/install | sh
-
-RUN echo '. /home/gitpod/.nix-profile/etc/profile.d/nix.sh' >> /home/gitpod/.bashrc
-RUN mkdir -p /home/gitpod/.config/nixpkgs && echo '{ allowUnfree = true; }' >> /home/gitpod/.config/nixpkgs/config.nix
-
-# Install cachix
-RUN . /home/gitpod/.nix-profile/etc/profile.d/nix.sh \
-  && nix-env -iA cachix -f https://cachix.org/api/v1/install \
-  && cachix use cachix \
-  && cachix use digitallyinduced
```

Comment:
We still need to run `cachix use digitallyinduced` somewhere down below to speed up the build

Reply:
Thanks for pointing that out 🙏 
I'll put it back.